index.html

```diff
@@ -114,7 +114,36 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
-
+ <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+ <div class="card">
+  <p class="name">Andrew Coote</p>
+  <p class="contact">
+    <i class="fab fa-twitter"></i>
+    <a href="https://twitter.com/Andrew_Coote" target="_blank">@Andrew_Coote</a>
+  </p>
+  <p class="about">Junior doctor with an interest in programming and informatics</p>
+  <div class="resources">
+    <p>3 Useful Dev Resources</p>
+    <ul>
+      <li>
+        <a href="https://www.theodinproject.com/" target="_blank" title="First Resource">
+          The Odin Project
+        </a>
+      </li>
+      <li>
+        <a href="https://www.codecademy.com/" target="_blank" title="Second Resource">
+          Codecademy
+        </a>
+      </li>
+      <li>
+        <a href="https://www.w3schools.com/" target="_blank" title="Third resource">
+          W3 schools
+        </a>
+      </li>
+    </ul>
+  </div>
+</div>
+<!-- ________*END TEMPLATE* Contributor card END ________  -->
```

Comment:
Please change this comment as well. 
`<!-- ________*END TEMPLATE* Contributor card END ________ -->` 
to `<!-- ________ Your_Name Contributor card END ________ -->`